dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -5,30 +5,32 @@ class EvaluateRubricJob < ApplicationJob
   queue_as :default
 
   S3_AI_BUCKET = 'cdo-ai'.freeze
+  # The path to the release directory in S3 which contains the AI rubric evaluation.
+  # When launching ai config changes, this path should be updated to the new release.
+  S3_AI_RELEASE_PATH = 'teaching_assistant/releases/2024-02-01-ai-rubrics-pilot-baseline/'.freeze
   STUB_AI_PROXY_PATH = '/api/test/ai_proxy'.freeze
 
-  # 2D Map from unit name and level name, to the name of the lesson files in S3
-  # which will be used for AI evaluation.
-  # TODO: This is a temporary solution. After the pilot, we should at least make
-  # the S3 pointer editable on levelbuilder, and eventually make all of the data
-  # it points to editable there too.
+  # 2D Map from unit name and level name, to the name of the lesson files within
+  # the release dir in S3 which will be used for AI evaluation.
   UNIT_AND_LEVEL_TO_LESSON_S3_NAME = {
```

Comment:
what about potentially moving this to S3? or does that make it too hard to test or experiment locally? maybe a hybrid.

Reply:
this is an interesting idea and I'm not sure where the best place for it is. one observation is that if we move this config to S3, then we lose some of the test coverage this PR adds which would ensure that any new lesson dirs added to this map have all the necessary files in the releases dir. so my inclination is there might need to be a compelling reason to move it there to overcome that?

for some context, where I think we want this to end up is in https://github.com/code-dot-org/aiproxy/blob/main/lib/assessment/config.py . then dashboard would just tell aiproxy what lesson to evaluate, and any updates to the latest release pointer could more easily be covered by the accuracy regression testing functionality which Ceara has added to the aiproxy repo. so, separately I am curious how that part of the plan sounds?